apps/frontend/src/app/3_organisms/components/TransactionGas/TransactionGas.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+
+import { AmountInput } from '@sovryn/ui';
```

Comment:
Move `apps/frontend/src/app/3_organisms/components` folder to `apps/frontend/src/app/3_organisms/TransactionStep/components` and update any imports